fixtures/etc_host.yml

```
@@ -0,0 +1,8 @@
+---
+- name: Set up etc_hosts for each host
+  become: true
+  hosts:
+    - quadlet
+    - client
```

Should this apply on all hosts and let Vagrant figure it out?
```suggestion
  hosts: all
```